RedHat-UX/red-hat-design-system
# docs(color): update status color table

## What I did

1. Updated the status table on the Color page to have the correct status names and colors. 


## Testing Instructions

1. Check that the token names and colors in the [status table](https://deploy-preview-1929--red-hat-design-system.netlify.app/foundations/color/usage/#status) matches what's shown in our [status tokens](https://red-hat-design-tokens.netlify.app/#color-status).

## Notes to Reviewers
